src/core/browser/qgsdataitem.h

```diff
@@ -291,7 +291,12 @@ class CORE_EXPORT QgsDataItem : public QObject
      *
      * \see capabilities2()
      */
-    virtual void setCapabilities( Qgis::BrowserItemCapabilities capabilities ) { mCapabilities = capabilities; }
+    virtual void setCapabilities( Qgis::BrowserItemCapabilities capabilities ) SIP_PYNAME( setCapabilitiesV2 ) { mCapabilities = capabilities; }
```

Comment:
```suggestion
    virtual void setCapabilities( Qgis::BrowserItemCapabilities capabilities ) SIP_PYNAME( setBrowserItemCapabilities ) { mCapabilities = capabilities; }
```

Just to avoid the ugly suffix

Reply:
I'll leave it for now -- this particular piece of api is a mess because it's (incorrectly??) marked as virtual.